.github/workflows/lint.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       - name: Installation
         uses: bahmutov/npm-install@v1
         with:
-         install-command: yarn install
-         # install-command: yarn install --immutable # Fails if yarn.lock is modified  (unfortunately only works for Yarn 2, and --frozen-lockfile is not the same!)
+          install-command: yarn install
+          # install-command: yarn install --immutable # Fails if yarn.lock is modified  (unfortunately only works for Yarn 2, and --frozen-lockfile is not the same!)
```

Comment:
Sorry, looks like autoformatter in editor inserted some whitespace :(

Reply:
looks fine anyway, as the rest use 2-space indentation already we can keep this